### ISSUE TITLE
Does not prompt to install new downloaded update automatically 

### ISSUE THREAD

**Issue by horpeyloaded** [esteemapp/esteem-surfer#130]
After it gave me pop up to install the latest version, I clicked on **Download** and it Downloaded but instead of opening the new update for me to install it stuck at 100%.  **The screenshot below;**
![2018-07-19 00_10_32-esteem surfer](https://user-images.githubusercontent.com/34909705/42913679-17192646-8aee-11e8-82a7-aafbaedcbbb8.png)

I had to navigate to my `\AppData\Roaming\eSteem Surfer` directory for me to see the new update and installed it manually **The screenshot below;**
![2018-07-19 00_11_50-esteem surfer](https://user-images.githubusercontent.com/34909705/42913607-cd8a8e52-8aed-11e8-9bea-9622c7573594.png)

***I noticed this on 1.0.6-1.0.7, 1.0.7-1.0.8 and now 1.0.8-1.0.9 hope it got fixed***
> after it has completed the update download, it should bring a new Toast/Popup if the user wants to *install now or later*. And if the user selects *later*, it should give tips about the directory to find the newly downloaded update to install because some casual users won't know where to find it later on.  

> also an option for users; *delete installation file after install or not* for free space

> tips to ask users to delete previous downloaded update in their *Appdata* directory like 1.0.6,1.0.7 etc for free space, or if there could be an option to do that with Surfer automatically.
### Occurred on  Windows 10

**Comment by talhasch** [comment]
just tested on both win7 and win10. worked without problem @horpeyloaded 

<img width="1364" alt="win7" src="https://user-images.githubusercontent.com/1177676/43538421-ab9ea0c0-95ca-11e8-9579-df76f790cace.png">

<img width="1166" alt="win10" src="https://user-images.githubusercontent.com/1177676/43538423-abc5fada-95ca-11e8-8283-2913cf8460e7.png">

**Comment by talhasch** [comment]
for next release i will add an error handler to update so we can figure out whats wrong if an upload error occur.

**Comment by fildunsky** [comment]
It works fine on my Windows 10 machine as well...

**Comment by talhasch** [comment]
do you have antivirus or some software which can block applications on your pc? @horpeyloaded

**Comment by horpeyloaded** [comment]
no, I even disabled Windows Defender @talhasch beacuse of patches/cracks I use. Let me try to update to the latest version tonight if the update error occurs again.

**Comment by fildunsky** [comment]
We are praying for you, @horpeyloaded

**Comment by fildunsky** [comment]
My Surfer on Windows says there is an update, I press to `Download` it. After downloading `Restart` button appears and after I press it Surfer closes itself and starts the update process automatically (with no prompt or warnings). Then it restarts as a new version.

I don't really know where it saves the update. Maybe @talhasch will tell.

**Comment by horpeyloaded** [comment]
Yes! works perfect, I will try it out.
For the installer file, you can use this command in **Run** start+r, input %appdata% Enter, then click on `eSteem Surfer` Folder, the installer may be here. @dunsky

**Comment by fildunsky** [comment]
Yes, I have all the updates there!
`C:\Users\myprecioususerfolderchangednameforparanoidsequrity\AppData\Roaming\eSteem Surfer` where `myprecioususerfolderchangednameforparanoidsequrity` is current logged in user

**Comment by horpeyloaded** [comment]
I think it would be better to change the updates file save directory to Download folder, so users can delete the old updates to save spaces. if you check your eSteem surfer directory `C:\Users\my-precious-user-folder-changed-name-for-paranoid-sequrity\AppData\Roaming\eSteem Surfer` @dunsky you may have tons of old updated software file of Surfer!

**Comment by talhasch** [comment]
@horpeyloaded stil having this issue with v2?

**Comment by horpeyloaded** [comment]
@talhasch No, it's fixed. It worked on v2.0.1 to v2.0.2 seamlessly on my PC. Thanks!
